theme_5/bourbon/addons/_triangle.scss

```diff
@@ -0,0 +1,63 @@
+@mixin triangle($size, $color, $direction) {
+  $width: nth($size, 1);
+  $height: nth($size, length($size));
+  $foreground-color: nth($color, 1);
+  $background-color: if(length($color) == 2, nth($color, 2), transparent);
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.